multi-config/values.yaml

```
@@ -0,0 +1,4 @@
+vector-aggregator:
+  extraConfigDirSources:
+    - configMap:
+        name: configs
```

For the new chart this would be...

```suggestion
vector:
  existingConfigMaps:
    - configs
```